dev/archery/archery/integration/datagen.py

```diff
@@ -179,8 +179,8 @@ def generate_column(self, size, name=None):
         return self.generate_range(size, lower_bound, upper_bound, name=name)
 
     def generate_range(self, size, lower, upper, name=None):
-        values = [int(x) for x in
-                  np.random.randint(lower, upper, size=size)]
+        values = list(map(int if self.bit_width < 64 else str,
+                          np.random.randint(lower, upper, size=size)))
```

Comment:
Append the extreme values so we test all integrations.

Reply:
alright